aplus/middleware.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
```

Comment:
The `lib/` is the location for files like these, even though the folder is badly named.